tests/test_lithium.py

```diff
@@ -697,6 +697,30 @@ def interesting(sub, conditionArgs, tempPrefix):  # pylint: disable=no-self-argu
                 else:
                     self.assertEqual(f.read(), valid_reductions[-1])
 
+    def test_minimize_collapse_braces(self):
```

Comment:
This should also test the case where removing the brackets makes it not interesting.

Reply:
Do you want this as a separate test entirely or just add a second interestingness class to the existing test?

Reply:
Added